src/CSModule.sol

```diff
@@ -1047,20 +1047,21 @@ contract CSModule is
     function settleELRewardsStealingPenalty(
         uint256[] calldata nodeOperatorIds
     ) external onlyRole(SETTLE_EL_REWARDS_STEALING_PENALTY_ROLE) {
+        ICSAccounting _accounting = accounting;
```

Comment:
How much does this move save in terms of gas? Just curious

Reply:
### Before

| src/CSModule.sol:CSModule contract      |                 |        |        |        |         |
|-----------------------------------------|-----------------|--------|--------|--------|---------|
| Deployment Cost                         | Deployment Size |        |        |        |         |
| 5285863                                 | 24987           |        |        |        |         |
| Function Name                           | min             | avg    | median | max    | # calls |
| settleELRewardsStealingPenalty          | 24494           | 82423  | 70685  | 150775 | 13      |

### After

| src/CSModule.sol:CSModule contract      |                 |        |        |        |         |
|-----------------------------------------|-----------------|--------|--------|--------|---------|
| Deployment Cost                         | Deployment Size |        |        |        |         |
| 5286739                                 | 24991           |        |        |        |         |
| Function Name                           | min             | avg    | median | max    | # calls |
| settleELRewardsStealingPenalty          | 24494           | 82395  | 70503  | 150287 | 13      |